mentoring/README.md

```diff
@@ -1,13 +1,8 @@
 ---
-Title: Kubernetes Upstream Mentoring Programs
----
-
-# Kubernetes Upstream Mentoring Programs
-
-This folder will be used for all mentoring initiatives for contributing to
-Kubernetes. Check out KubeCon and other CNCF events and programs for end user
-training and learning.
```

Comment:
Since this will now be published in both locations it might be better to reword it slightly:

```
This page indexes all of the mentoring initiatives for contributing to the
Kubernetes project. For end user mentoring initiatives, check out KubeCon
and other CNCF events and programs.
```